docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
I'm not really maintaining redux-saga-test anymore. @aaronjensen is maintaining it mostly. Maybe this org is interested in taking it over? I'm also happy to just refer people to redux-saga-test-plan.